src/angular/projects/spark-angular/src/lib/components/sprk-pagination/sprk-pagination.component.ts

```diff
@@ -12,7 +12,6 @@ import {
     <nav
       aria-label="Pagination Navigation"
       [ngClass]="getClasses()"
-      *ngIf="paginationType === 'default'"
       [attr.data-id]="idString"
     >
       <sprk-unordered-list
```

Comment:
Pagination feels like it would imply that order matters. Do we think this should be an ordered list instead?

Reply:
I just confirmed that this does *not* affect VoiceOver behavior. I do think it makes more sense semantically though.